### PR TITLE
MAUI-1677 Updated the UG review changes for MAUI SfEffectsView

### PR DESCRIPTION
**Pull request by Kumaraguru1081**
**Description**

Updated the UG review changes for MAUI SfEffectsView.

### CODE REVIEW

File: MAUI/Effects-View/Effects/Highlight.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: The .NET MAUI Effects View control | Syncfusion
```
Comment (Choza-rajan) [inline MAUI/Effects-View/Effects/Highlight.md:3]: The .NET MAUI Highlight Effects | Effects View control | Syncfusion

File: MAUI/Effects-View/Effects/Ripple.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+layout: post
+title: The .NET MAUI Effects View control | Syncfusion
```
Comment (Choza-rajan) [inline MAUI/Effects-View/Effects/Ripple.md:3]: The .NET MAUI Ripple Effects | Effects View control | Syncfusion

File: MAUI/Effects-View/Effects/Rotation.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+---
+layout: post
+title: The .NET MAUI Effects View control | Syncfusion
```
Comment (Choza-rajan) [inline MAUI/Effects-View/Effects/Rotation.md:3]: The .NET MAUI Rotate Animation Effects | Effects View control | Syncfusion

File: MAUI/Effects-View/Effects/Scale.md [hunk 4]
```diff
@@ -0,0 +1,41 @@
+---
+layout: post
+title: The .NET MAUI Effects View control | Syncfusion
```
Comment (Choza-rajan) [inline MAUI/Effects-View/Effects/Scale.md:3]: The .NET MAUI Scale Animation Effects | Effects View control | Syncfusion

File: MAUI/Effects-View/Effects/Selection.md [hunk 5]
```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: The .NET MAUI Effects View control | Syncfusion
```
Comment (Choza-rajan) [inline MAUI/Effects-View/Effects/Selection.md:3]: The .NET MAUI Selection Effects | Effects View control | Syncfusion

File: MAUI/Effects-View/Effects/Combinations.md [hunk 6]
```diff
@@ -0,0 +1,131 @@
+---
+layout: post
+title: The .NET MAUI Effects View control | Syncfusion
```
Comment (Choza-rajan) [inline MAUI/Effects-View/Effects/Combinations.md:3]: The .NET MAUI Combination Effects | Effects View control | Syncfusion

File: MAUI/Effects-View/Effects/Highlight.md [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+{
+    TouchDownEffects = SfEffects.Highlight,
+    HighlightBackground = new SolidColorBrush(Colors.Aqua)
+};
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Effects View highlight effect](Effects_images/EffectsView_Highlight.png)
```
Comment (Choza-rajan) [inline MAUI/Effects-View/Effects/Highlight.md:37]: .net maui highlight effect

File: MAUI/Effects-View/Effects/Ripple.md [hunk 8]
```diff
@@ -0,0 +1,34 @@
...
+var effectsView = new SfEffectsView()
+{
+    TouchDownEffects = SfEffects.Ripple
+};
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Effects View ripple effect](Effects_images/EffectsView_Ripple.gif)
```
Comment (Choza-rajan) [inline MAUI/Effects-View/Effects/Ripple.md:34]: .net maui ripple effect

File: MAUI/Effects-View/Effects/Rotation.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+var effectsView = new SfEffectsView
+{
+    TouchDownEffects = SfEffects.Rotation,
+    Angle = 180,
+};
+
+{% endhighlight %}
+
+{% endtabs %}
```
Comment (Choza-rajan) [inline MAUI/Effects-View/Effects/Rotation.md:35]: Add the gif or image for rotation animation

File: MAUI/Effects-View/Effects/Scale.md [hunk 10]
```diff
@@ -0,0 +1,41 @@
...
+    TouchDownEffects = SfEffects.None,
+    TouchUpEffects = SfEffects.None,
+    LongPressEffects = SfEffects.Scale
+};
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Effects View scale effect](Effects_images/EffectsView_Scale.png)
```
Comment (Choza-rajan) [inline MAUI/Effects-View/Effects/Scale.md:41]: .net maui scale animation

File: MAUI/Effects-View/Effects/Selection.md [hunk 11]
```diff
@@ -0,0 +1,37 @@
...
+{
+    LongPressEffects = SfEffects.Selection,
+    SelectionBackground = new SolidColorBrush(Colors.Aqua)
+};
+
+{% endhighlight %}
+
+{% endtabs %}
+
+![.NET MAUI Effects View selection effect](Effects_images/EffectsView_Selection.png)
```
Comment (Choza-rajan) [inline MAUI/Effects-View/Effects/Selection.md:37]: .net maui selection effect

File: MAUI/Effects-View/Effects/Ripple.md (hunk 2 above)
Comment (Kumaraguru1081) [reply MAUI/Effects-View/Effects/Ripple.md]: Updated.

File: MAUI/Effects-View/Effects/Rotation.md (hunk 3 above)
Comment (Kumaraguru1081) [reply MAUI/Effects-View/Effects/Rotation.md]: Updated.

File: MAUI/Effects-View/Effects/Selection.md (hunk 11 above)
Comment (Kumaraguru1081) [reply MAUI/Effects-View/Effects/Selection.md]: Updated.

File: MAUI/Effects-View/Effects/Scale.md (hunk 10 above)
Comment (Kumaraguru1081) [reply MAUI/Effects-View/Effects/Scale.md]: Updated.

File: MAUI/Effects-View/Effects/Ripple.md (hunk 8 above)
Comment (Kumaraguru1081) [reply MAUI/Effects-View/Effects/Ripple.md]: Updated.

File: MAUI/Effects-View/Effects/Highlight.md (hunk 1 above)
Comment (Kumaraguru1081) [reply MAUI/Effects-View/Effects/Highlight.md]: Updated.

File: MAUI/Effects-View/Effects/Scale.md (hunk 4 above)
Comment (Kumaraguru1081) [reply MAUI/Effects-View/Effects/Scale.md]: Updated.

File: MAUI/Effects-View/Effects/Highlight.md (hunk 7 above)
Comment (Kumaraguru1081) [reply MAUI/Effects-View/Effects/Highlight.md]: Updated.

File: MAUI/Effects-View/Effects/Combinations.md (hunk 6 above)
Comment (Kumaraguru1081) [reply MAUI/Effects-View/Effects/Combinations.md]: Updated.

File: MAUI/Effects-View/Effects/Selection.md (hunk 5 above)
Comment (Kumaraguru1081) [reply MAUI/Effects-View/Effects/Selection.md]: Updated.

File: MAUI/Effects-View/Effects/Rotation.md (hunk 9 above)
Comment (Kumaraguru1081) [reply MAUI/Effects-View/Effects/Rotation.md]: Added the GIF for rotation animation.

**Review by PaulAnderson02 — APPROVED** [review]
Fine

**Pull request merged**